Mathlib/Algebra/Polynomial/Lifts.lean

```diff
@@ -227,4 +227,21 @@ theorem monic_of_monic_mapAlg [FaithfulSMul R S] {p : Polynomial R} (hp : (mapAl
 
 end Algebra
 
+section IsScalarTower
+
+variable {A : Type*} (B C : Type*) [CommSemiring A] [CommSemiring B] [Semiring C]
```

Comment:
Those have not much to do with `Polynomial.Lifts`, right? What about moving them to another file?

Reply:
Ah, I see. Can you have a look to see if it possible to move it just after [Polynomial.aeval](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Polynomial/AlgebraMap.html#Polynomial.aeval)? Thanks!